src/monodroid/jni/monodroid-glue.c

```diff
@@ -3361,13 +3361,29 @@ set_profile_options (JNIEnv *env)
 	if (!output) {
 		const char* col = strchr (value, ':');
 		char *ovalue;
+		char *extension;
 
-		output = path_combine (override_dirs [0], "profile.mlpd");
+		if (col && !strncmp (value, "log:", 4)
+		    || !strncmp (value, "log", 3))
```

Comment:
Code style: please place the `||` on the previous line, please indent this line an added tab stop, and please use braces:

```c
if (col && !strncmp (value, "log:", 4) ||
        !strncmp (value, "log", 3)) {
    extension = "mlpd";
} else ...
```